### ISSUE TITLE
[BITV] 9.2.1.1/7.2 - Accordion elements of type "Error" within the table are not focusable and activatable using the keyboard. (1)

### ISSUE THREAD

**Issue by AndyScherzinger** [nextcloud/server#36984]
- [ ] Fix
- [ ] Backport

<img width="1331" alt="524608540c9af27f20ffb87c027c6326" src="https://user-images.githubusercontent.com/1315170/222246080-188244e3-147c-49b5-a69b-244f017c0b4a.png">

![414c728086658dfdb595a873bd04deeb](https://user-images.githubusercontent.com/1315170/222246049-8429c737-c23e-4280-8756-9ef709854d4a.png)

<details>
<summary>Details</summary>

https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-6bd94ebc49-v7-n2

</details>

**Comment by szaimen** [comment]
I think this was fixed with the logreader rewrite right @susnux?

**Comment by susnux** [comment]
yes
